core/src/main/java/com/microsoft/applicationinsights/internal/util/Sanitizer.java

```diff
@@ -168,7 +187,57 @@ private static String MakeKeyNonEmpty(String key) {
         return key;
     }
 
+    @Obsolete
     private static String truncate(String value, int maxLength) {
         return value.length() > maxLength ? value.substring(0, maxLength) : value;
     }
+
+    @Obsolete
```

Comment:
That's an agile development, we add deprecated methods :)

Reply:
The reason it is still there is because if in turn someone uses this classes public method which in theory they shouldn't this will atleast makes things right. If you want I can modify those public methods and remove the usage of this method and then remove this method totally. Let me know on that.